arm_robots/scripts/debugging/victor_debugging.py

```diff
@@ -0,0 +1,100 @@
+"""
+This script is similar to victor_basic_motion, but offers a slightly easier starting point for debugging
+"""
+
+# ! /usr/bin/env python
+import colorama
+import numpy as np
+import roscpp_initializer
+import rospy
+from geometry_msgs.msg import Pose
+from tf.transformations import quaternion_from_euler
+
+from arc_utilities import ros_init
+from arm_robots.victor import Victor
+from victor_hardware_interface_msgs.msg import ControlMode
+
+ask_before_moving = False
+
+
+def myinput(msg):
+    global ask_before_moving
+    if ask_before_moving:
+        input(msg)
+    else:
+        print(msg)
+
+
+def open_and_close_grippers(victor):
+    rospy.sleep(1)
+    victor.open_left_gripper()
+    rospy.sleep(1)
+    victor.close_left_gripper()
+    rospy.sleep(1)
+    victor.open_right_gripper()
+    rospy.sleep(1)
+    victor.close_right_gripper()
+    rospy.sleep(1)
+
+
+@ros_init.with_ros("victor_basic_motion")
+def main():
+    np.set_printoptions(suppress=True, precision=0, linewidth=200)
+    colorama.init(autoreset=True)
+
+    victor = Victor(display_goals=False)
```

Comment:
if display goals did in fact get fixed as I think you mentioned, consider making this True

Reply:
When running I get the warning: 
[WARN][/victor_debugging]: failed to connect publishers /display_robot_state/joint_config_goal

What is suppose to be listening to this topic? Presumably something in RViz? I'll add it to the `victor.rviz`